RecoPixelVertexing/PixelTrackFitting/interface/RiemannFit.h

```diff
@@ -3,6 +3,9 @@
 
 #include "FitResult.h"
 
+#ifndef RFIT_DEBUG
+#define RFIT_DEBUG 0
+#endif  // RFIT_DEBUG
```

Comment:
Is this `#define` something to be kept? (just asking)

Reply:
it seems to be still used also in the final version, #338 .

Reply:
Ciao Matti,
I still think this `#define` is useful. There are a few aspects that should be better understood in both fits and having the ability to quickly dump information is valuable, to me.

Reply:
That's fine. I just wanted confirm that having some debug stuff turned on by default is intentional.

Reply:
Hi Matti,
why do you say

> turned on by default

?

Reply:
These lines defines `RFIT_DEBUG` macro if it was not defined before, i.e. the macro is always defined. Later in the file only the existence of the macro is checked with `#ifdef RFIT_DEBUG`.

Reply:
Ah... I assumed we were using `#if RFIT_DEBUG`, not `#ifdef RFIT_DEBUG`.